### PR TITLE
[WIP] Remove unused translatins

### PR DESCRIPTION
**Pull request by Serhy**
Removed not all captions found by the script because I doubt they may not be used (or they have short translation which may be re-used in any place)

Kept  
```

ens-username-owned-continue
status-tx-not-found
puk-and-pairing-codes-displayed
write-down-and-store-securely
group-chat-admin-added
biometric-auth-reason-verify
command-button-send
send-request-amount-max-decimals
finishing-card-setup
incorrect-code
send-sending-to
show-qr
card-is-blank
decryption-failed-content
unknown-status-go-error
invalid-range
available-participants
group-chat-created
invalid-key-content
complete-hardwallet-setup
enter-12-words
no-pairing-slots-available
pairing-card
add-account-incadd-to-contacts-orrect-password
group-chat-name-changed
p-title
ens-show-username
copy-qr
copy-info
group-chat-member-removed
group-chat-member-joined
transaction-request
invalid-key-confirm
send-request-unknown-token
ask-in-status
group-chat-member-added
close-app-content
to-block
migrations-failed-content
to-see-this-message
active-unknown

```
Reason is if I remove everything in one go then it will require much more work to return back appropriate caption. So this is first iteration for now

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -17,9 +17,8 @@
     "add-a-watch-account": "Add a watch-only address",
     "add-account": "Add account",
     "add-account-description": "You can import any type of Ethereum account to add it to your Status wallet",
-    "add-account-incorrect-password": "Password seems to be incorrect. Enter the password you use to unlock the app.",
+    "add-account-incadd-to-contacts-orrect-password": "Password seems to be incorrect. Enter the password you use to unlock the app.",
```
Comment (flexsurfer) [inline translations/en.json]: @Serhy smth wrong here

File: translations/en.json [hunk 2]
```diff
@@ -148,17 +137,11 @@
     "clear-history-title": "Clear history?",
     "close-app-button": "Confirm",
     "close-app-content": "The app will stop and close. When you reopen it, the selected network will be used",
-    "close-app-title": "Warning!",
+    "close-apextensions-network-not-found\nextensions-network-not-found\np-title": "Warning!",
```
Comment (flexsurfer) [inline translations/en.json]: and here

File: translations/en.json (hunk 1 above)
Comment (Serhy) [reply translations/en.json]: Fixed

File: translations/en.json (hunk 2 above)
Comment (Serhy) [reply translations/en.json]: Fixed

**Review by jinhojang6 — APPROVED** [review]

**Comment by flexsurfer** [comment]
@Serhy could  you resolve conflicts,  and merge it?

**Comment by flexsurfer** [comment]
@Serhy could you resolve conflicts, and merge it?

**Comment by churik** [comment]
fixes https://github.com/status-im/status-react/issues/8698

**Comment by Serhy** [comment]
Merging!

**Pull request merged**